pkg/restclient/common.go

```diff
@@ -114,6 +123,8 @@ func (c *Client) do(method string, url string, data interface{}, expectStatusCod
 		req.Header.Set("Authorization", "bearer "+getAuthToken(c.username, c.password))
 	}
 
+	c.sendOriginArgs(req)
```

Comment:
Suggestion: this can be named to `setOriginArgs`

Reply:
We have a `c.SetOriginArgs` which does a different job to `sendOriginArgs`. The suggested rename makes the code more confusing.

I'd suggest that it be renamed to  `addOriginArgs`, but that would take away the consistent naming with the other "set" methods being called. Maybe also rename 'c.setAgent` to `c.addAgent`?

Reply:
Perhaps we keep SetOriginArgs and rename this to addOriginArgsToHeaders ?